### DISCUSSION TITLE
Filter notifications by title

### DISCUSSION THREAD

**Discussion by sethfri** [community/community#3660] [Mobile] [answered]
My company uses a bot to automate dependency upgrades and send out PRs when they fail. For most repos I'm subscribed to, I don't care about these and only want to get notifications for human-created PRs.

It would be great if I could say "don't send me notifications for PRs _in this repo_ with X in the title".

Thank you!

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion here @sethfri – same feeling from me, I get a ton of Dependabot notifications every week 😛 

I'll flag this for our team (+ the notifications team). Perhaps a rules-based system could be cool to let people set up on their own, or we could be more conservative in how we handle bot-triggered events.

**Reply by sethfri** [reply]
Rules-based system would be great since I don't think our bot is a GitHub native bot!

**Reply by MainasuK** [reply]
The Crowdin GitHub integration has the same issue. Lots of commits in the "New Crowdin updates" PR every hour.

**Reply by sethfri** [reply]
Hi @brianlovin any update on this?

**Reply by brianlovin** [reply]
No updates right now 😊
